scapy/contrib/http2.py

```diff
@@ -192,7 +192,7 @@ def __init__(self, name, default, size):
         @return None
         @raise AssertionError
         """
-        assert(isinstance(default, types.NoneType) or (isinstance(default, (int, long)) and default >= 0))
+        assert(isinstance(default, type(None)) or (isinstance(default, (int, long)) and default >= 0))
```

Comment:
Maybe `isinstance(default, type(None))` could be rewritten as `default is None`?